devicemaps.json

```diff
@@ -1041,6 +1041,14 @@
         ],
         "lte": []
       },
+      "SSR1300-DC": {
+        "clei": "INMJ410HRA",
+        "part_number": "650-180066",
```

Comment:
Please correct the part number to 650-180064

Reply:
this is super confusing - as long as they put the right information in the EEPROM it doesn't matter to me. Thanks for catching it.